eZ/Publish/Core/Limitation/Tests/LanguageLimitationTypeTest.php

```diff
@@ -0,0 +1,467 @@
+<?php
+
+/**
+ * File containing a Test Case for LimitationType class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Limitation\Tests;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\Operator;
+use eZ\Publish\API\Repository\Values\User\Limitation;
+use eZ\Publish\API\Repository\Values\User\Limitation\LanguageLimitation;
+use eZ\Publish\API\Repository\Values\User\Limitation\ObjectStateLimitation;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\LanguageCode;
+use eZ\Publish\Core\Base\Exceptions\NotFoundException;
+use eZ\Publish\Core\Limitation\LanguageLimitationType;
+use eZ\Publish\Core\Repository\Values\Content\Location;
+use eZ\Publish\Core\Repository\Values\Content\ContentCreateStruct;
+use eZ\Publish\Core\Repository\Values\Content\ContentUpdateStruct;
+use eZ\Publish\SPI\Persistence\Content\Language\Handler as SPIHandler;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+
+/**
+ * Test Case for LimitationType.
+ */
+class LanguageLimitationTypeTest extends Base
+{
+    /**
+     * @var \eZ\Publish\SPI\Persistence\Content\Language\Handler|\PHPUnit\Framework\MockObject\MockObject
+     */
+    private $contentLanguageHandlerMock;
+
+    /**
+     * Setup Language Handler mock.
+     */
+    public function setUp()
+    {
+        parent::setUp();
+        $this->contentLanguageHandlerMock = $this->createMock(SPIHandler::class);
+    }
+
+    /**
+     * Tear down Language Handler mock.
+     */
+    public function tearDown()
+    {
+        unset($this->contentLanguageHandlerMock);
+        parent::tearDown();
+    }
+
+    /**
+     * @return \eZ\Publish\Core\Limitation\LanguageLimitationType
+     */
+    public function testConstruct()
+    {
+        return new LanguageLimitationType($this->getPersistenceMock());
```

Comment:
Doesn't this mark the test as risky? (no assertions).

Reply:
I think so to, if there are no assertions it should not be a test